python/pyspark/ml/classification.py

```diff
@@ -1327,8 +1327,6 @@ class MultilayerPerceptronClassifier(JavaEstimator, HasFeaturesCol, HasLabelCol,
                       "remaining data in a partition then it is adjusted to the size of this " +
                       "data. Recommended size is between 10 and 1000, default is 128.",
                       typeConverter=TypeConverters.toInt)
-    solver = Param(Params._dummy(), "solver", "The solver algorithm for optimization. Supported " +
```

Comment:
We still need to print out supported options for different algorithms, so could we override param definition in Python as well as what we did in Scala?